infra/modules/providers/azure/app-service/main.tf

```diff
@@ -0,0 +1,13 @@
+resource "azurerm_resource_group" "appsvc" {
+  name     = "${var.resource_group_name}"
+  location = "${var.resource_group_location}"
+  tags     = "${var.resource_tags}"
+}
+
+resource "azurerm_app_service" "appsvc" {
+  name                = "${var.app_service_name}"
+  location            = "${azurerm_resource_group.appsvc.location}"
+  resource_group_name = "${azurerm_resource_group.appsvc.name}"
+  app_service_plan_id = "${var.app_service_plan_id}"
```

Comment:
the `app_settings` and `site_config` are required for container deployment... How would a developer provide a container and `DOCKER_REGISTRY_SERVER_URL` to the provisioned app service resource?